### PR TITLE
fix table formatting in readme

### PR DESCRIPTION
**Pull request by arlimus**
Has been bugging me for a year now...

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## GitHub Users Who Are Not Authorized To Contribute

The following GitHub users do not appear to have signed a CLA:

* @arlimus

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

File: README.md [hunk 1]
```diff
@@ -386,15 +386,13 @@ It may be informative to look at what [tests Chef Delivery](https://github.com/c
...
-| **Author:**          | Dominik Richter (<drichter@chef.io>)
-
-| **Author:**          | Christoph Hartmann (<chartmann@chef.io>)
-
-| **Copyright:**       | Copyright (c) 2015 Chef Software Inc.
-
-| **Copyright:**       | Copyright (c) 2015 Vulcano Security GmbH.
-
-| **License:**         | Apache License, Version 2.0
+|  | License |
```
Comment (chris-rock) [inline README.md]: Is there a reason we need to have the first row?

Comment (arlimus) [reply README.md]: Yes: to create a working table in github

Comment (arlimus) [reply README.md]: Ie:

| table |
| line |

vs

| table |
| --- |
| line |

**Pull request merged**